cluster/config.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/minio/minio/pkg/env"
 )
 
+func getNs() string {
```

Comment:
sneaky

Reply:
since we are moving to controller model, i'm introducing the controller